### PR TITLE
Remove extra and inconsistent sentences for windows insider.

### LINKED ISSUE #29628: gRPC services: Update info on IIS
ITNOA

Please update documentation about IIS,

Windows 11 has 22621.1848 build number, that higher than Windows 10 Build 20300.1000

Windows Server 2022 has 20348.1787 build number, that higher than Windows 10 Build 20300.1000

So 

> which may require the use of a [Windows Insider](https://insider.windows.com/) build

sentence is extra, and must be removed from page
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4e072aff-35c1-1fc2-3f94-97dc7a809414
* Version Independent ID: 2f05b9da-0d50-ce84-d6af-332da7d9d625
* Content: [gRPC services with ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/grpc/aspnetcore?view=aspnetcore-7.0&tabs=visual-studio)
* Content Source: [aspnetcore/grpc/aspnetcore.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/grpc/aspnetcore.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-grpc**
* GitHub Login: @JamesNK
* Microsoft Alias: **jamesnk**

### PR DESCRIPTION
**Pull request by soroshsabz**
ITNOA

Fixes #29628 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/grpc/aspnetcore.md](https://github.com/dotnet/AspNetCore.Docs/blob/add580ee74c8416a9e08d3f20a879cdc3f1aaff2/aspnetcore/grpc/aspnetcore.md) | [gRPC services with ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/grpc/aspnetcore?branch=pr-en-us-29629) |

### CODE REVIEW

**Comment by wadepickett** [comment]
@soroshsabz, thanks for raising the issue and your contribution here.  There is a step for license/CLA to complete above as a first step.

**Comment by wadepickett** [comment]
@JamesNK,
Per https://github.com/dotnet/AspNetCore.Docs/pull/25349/files should it be the following?
IIS:  Windows 11 Build 20200.0000 or for IIS 
HTTP.sys: Windows 10 Build 19529

**Comment by soroshsabz** [comment]
> @soroshsabz, thanks very much for raising the issue and your contribution here. There is a step for license/CLA to complete as a required step.

@wadepickett I think approve cla before in many years ago, How to resign that? if you miss my sign.

**Comment by soroshsabz** [comment]
> @JamesNK, Per https://github.com/dotnet/AspNetCore.Docs/pull/25349/files should it be the following? IIS: Windows 11 Build 20200.0000 or for IIS HTTP.sys: Windows 10 Build 19529

@wadepickett I do not understand your meaning correctly, Did you mean change my sentences to that?

I think my sentences is better in this doc, because more simple and more general, but if you want, I can change it

**Comment by wadepickett** [comment]
> > @JamesNK, Per https://github.com/dotnet/AspNetCore.Docs/pull/25349/files should it be the following? IIS: Windows 11 Build 20200.0000 or for IIS HTTP.sys: Windows 10 Build 19529
> 
> @wadepickett I do not understand your meaning correctly, Did you mean change my sentences to that?
> 
> I think my sentences is better in this doc, because more simple and more general, but if you want, I can change it

@soroshsabz,
I was verifying with the author the exact versions required.  The question was not directed at you.  Sorry for the confusion.  I will wait for James to verify.  He is on a different time zone, so the response may be tomorrow.  Thanks.

**Comment by wadepickett** [comment]
> > @soroshsabz, thanks very much for raising the issue and your contribution here. There is a step for license/CLA to complete as a required step.
> 
> @wadepickett I think approve cla before in many years ago, How to resign that? if you miss my sign.

@soroshsabz,
There should be an option for the license/cla somewhere under the "Checks" section here.  If not, perhaps you can copy/paste to show what you are seeing there so I can understand that state.

image

**Comment by soroshsabz** [comment]
@wadepickett All check have passed at now (without any acting from me)

**Comment by wadepickett** [comment]
> @wadepickett All check have passed at now (without any acting from me)

Interesting.  The server must have been terribly slow.  Sorry that happened.

**Comment by wadepickett** [comment]
@JamesNK,  should this be Windows Build 20200.0000 or 20300.1000 as the requirement for IIS?

File: aspnetcore/grpc/aspnetcore.md [hunk 1]
```diff
@@ -193,13 +193,13 @@ For more information on enabling HTTP/2 and TLS with Kestrel, see [Kestrel endpo
 
 ## IIS
 
-[Internet Information Services (IIS)](xref:host-and-deploy/iis/index) is a flexible, secure and manageable Web Server for hosting web apps, including ASP.NET Core. .NET 5 and Windows 10 Build 20300.1000 or later are required to host gRPC services with IIS, which may require the use of a [Windows Insider](https://insider.windows.com) build.
+[Internet Information Services (IIS)](xref:host-and-deploy/iis/index) is a flexible, secure and manageable Web Server for hosting web apps, including ASP.NET Core. .NET 5 and Windows Build 20300.1000 or later are required to host gRPC services with IIS.
```
Comment (JamesNK) [inline aspnetcore/grpc/aspnetcore.md:196]:
```suggestion
[Internet Information Services (IIS)](xref:host-and-deploy/iis/index) is a flexible, secure and manageable Web Server for hosting web apps, including ASP.NET Core. .NET 5 and Windows 11 Build 22000 or Windows Server 2022 Build 20348 or later are required to host gRPC services with IIS.
```

File: aspnetcore/grpc/aspnetcore.md [hunk 2]
```diff
@@ -193,13 +193,13 @@ For more information on enabling HTTP/2 and TLS with Kestrel, see [Kestrel endpo
...
 
 IIS must be configured to use TLS and HTTP/2. For more information, see <xref:host-and-deploy/iis/protocols>.
 
 ## HTTP.sys
 
-[HTTP.sys](xref:fundamentals/servers/httpsys) is a web server for ASP.NET Core that only runs on Windows. .NET 5 and Windows 10 Build 19529 or later are required to host gRPC services with HTTP.sys, which may require the use of a [Windows Insider](https://insider.windows.com) build.
+[HTTP.sys](xref:fundamentals/servers/httpsys) is a web server for ASP.NET Core that only runs on Windows. .NET 5 and Windows Build 19529 or later are required to host gRPC services with HTTP.sys.
```
Comment (JamesNK) [inline aspnetcore/grpc/aspnetcore.md:202]:
```suggestion
[HTTP.sys](xref:fundamentals/servers/httpsys) is a web server for ASP.NET Core that only runs on Windows. .NET 5 and Windows 11 Build 22000 or Windows Server 2022 Build 20348 or later are required to host gRPC services with HTTP.sys.
```

**Review by JamesNK — COMMENTED** [review]
Makes this page consistent with supported-platforms.md: https://learn.microsoft.com/en-us/aspnet/core/grpc/supported-platforms?view=aspnetcore-7.0#supported-aspnet-core-servers

**Review by wadepickett — APPROVED** [review]
Approve with changes from JamesNK.  

@soroshsabz, thanks again for noting the issue and making a contribution.  Very appreciated.

**Review by JamesNK — APPROVED** [review]

**Comment by wadepickett** [comment]
This change was only set on an early version, also putting the requirement changes on all versions as was done for the example topic we are following here...

File: aspnetcore/grpc/aspnetcore.md [hunk 3]
```diff
@@ -86,10 +86,8 @@ gRPC services can be hosted by all built-in ASP.NET Core servers.
 > * IIS&dagger;
 > * HTTP.sys&Dagger;
 
-&dagger;IIS requires .NET 5 and Windows 10 Build 20300.1000 or later.  
-&Dagger;HTTP.sys requires .NET 5 and Windows 10 Build 19529 or later.
-
-The preceding Windows 10 Build versions may require the use of a [Windows Insider](https://insider.windows.com) build.
+&dagger;IIS requires .NET 5 and Windows 11 Build 22000 or Windows Server 2022 Build 20348 or later are required to host gRPC services with IIS.
```
Comment (wadepickett) [inline aspnetcore/grpc/aspnetcore.md:89]:
```suggestion
&dagger;IIS requires .NET 5 and Windows 11 Build 22000 or Windows Server 2022 Build 20348 or later to host gRPC services.  
```

File: aspnetcore/grpc/aspnetcore.md [hunk 4]
```diff
@@ -86,10 +86,8 @@ gRPC services can be hosted by all built-in ASP.NET Core servers.
 > * IIS&dagger;
 > * HTTP.sys&Dagger;
 
-&dagger;IIS requires .NET 5 and Windows 10 Build 20300.1000 or later.  
-&Dagger;HTTP.sys requires .NET 5 and Windows 10 Build 19529 or later.
-
-The preceding Windows 10 Build versions may require the use of a [Windows Insider](https://insider.windows.com) build.
+&dagger;IIS requires .NET 5 and Windows 11 Build 22000 or Windows Server 2022 Build 20348 or later are required to host gRPC services with IIS.  
+&Dagger;HTTP.sys requires .NET 5 and Windows 11 Build 22000 or Windows Server 2022 Build 20348 or later are required to host gRPC services with HTTP.sys.
```
Comment (wadepickett) [inline aspnetcore/grpc/aspnetcore.md:90]:
```suggestion
&Dagger;HTTP.sys requires .NET 5 and Windows 11 Build 22000 or Windows Server 2022 Build 20348 or later to host gRPC services.
```

**Pull request merged**